tensorflow/lite/micro/kernels/xtensa/reduce.cc

```diff
@@ -0,0 +1,374 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
```

Comment:
@tingyan19 This file shares a considerable amount of code with `tensorflow/lite/micro/kernels/reduce.cc`. Is there an easier way to share code between the two implementations? Could some of it be moved to `tensorflow/lite/micro/kernels/reduce_common.cc`?

Reply:
@rascani I think it's a good idea. Let's merge this version for now. I have created a bug to track the effort to refactor this.